README.md

```diff
@@ -0,0 +1,60 @@
+# Shavar list testing using test pages
+Instructions on how to create test profiles: https://wiki.mozilla.org/Services/TrackingProtection/Shavar_Server_-_Testing#SET_CUSTOM_PREFS
+
+### https://mozilla.github.io/tracking-test/ads.html
+1. Create a BASE profile
```

Comment:
This explanation for how to test profiles is very clear.  Do you think we should also add an explanation of what these profiles are used for (for context)?  I recall that was particularly confusing before (i.e. why do we have these pref sets definined in the prefs.ini).  For example, while it's clear from the test name what each page testing, it seems like we should also provide some brief summary of why we use BASE for certain tests, STRICT for others and none for others, so our test methodology is clear.

Reply:
I think an explanation of what the lists are verifying should be defined elsewhere, in developer documentation. It should be clear that the test profiles are for testing specific lists, and that is shown in their naming convention. Our test methodology should cover all of the list variations and set up steps more than explaining why they are that way [imo].